lib/openstudio-standards.rb

```diff
@@ -437,6 +437,7 @@ module OpenstudioStandards
   require_relative "#{proto}/ashrae_90_1/ashrae_90_1_2019/ashrae_90_1_2019.hvac_systems"
   require_relative "#{proto}/ashrae_90_1/ashrae_90_1_2019/ashrae_90_1_2019.AirTerminalSingleDuctVAVReheat"
   require_relative "#{proto}/ashrae_90_1/ashrae_90_1_2019/ashrae_90_1_2019.PumpVariableSpeed"
+  require_relative "#{proto}/ashrae_90_1/ashrae_90_1_2019/ashrae_90_1_2019.HeatExchangerAirToAirSensibleAndLatent"
```

Comment:
A new ruby file was created about heat exchanger calculation for ASHRAE 90.1-2019. Thus, the require_relative for this file was added.